boards/Makefile.common

```diff
@@ -10,7 +10,7 @@ export RUSTUP_TOOLCHAIN ?= nightly-2018-03-07
 TOOLCHAIN ?= arm-none-eabi
 
 XARGO ?= xargo
-MINIMUM_XARGO_VERSION ?= 0.3.8
+MINIMUM_XARGO_VERSION ?= 0.3.08
```

Comment:
It this intentional?

Reply:
This was intentional. Though I am not the one that originally made the change - I think it was Connor - I recall that he was running xargo version 0.3.10, and yet was getting a warning that his version of xargo was less than the required minimum xargo version. This change fixed the problem.

Reply:
Ok, I don't think it matter to be honest we will soon get rid off xargo once https://github.com/tock/tock/pull/881 is merged!